.github/scripts/unittest.sh

```diff
@@ -4,15 +4,11 @@ set -euo pipefail
 
 ./.github/scripts/setup-env.sh
 
-# Prepare conda
-CONDA_PATH=$(which conda)
-eval "$(${CONDA_PATH} shell.bash hook)"
-conda activate ci
+# Activate conda environment
+eval "$($(which conda) shell.bash hook)" && conda deactivate && conda activate ci
 
 echo '::group::Install testing utilities'
 pip install --progress-bar=off pytest pytest-mock pytest-cov
 echo '::endgroup::'
 
-echo '::group::Run unittests'
```

Comment:
After a command is finished, GH automatically collapses groups. Since we want to see the tests, drop the grouping here.